static/app/views/onboarding/targetedOnboarding/components/firstEventFooter.tsx

```diff
@@ -0,0 +1,200 @@
+import Link from 'sentry/components/links/link';
+import {IconCheckmark} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import pulsingIndicatorStyles from 'sentry/styles/pulsingIndicator';
+import space from 'sentry/styles/space';
+import {Group, Organization, Project} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import EventWaiter from 'sentry/utils/eventWaiter';
+import testableTransition from 'sentry/utils/testableTransition';
+import CreateSampleEventButton from 'sentry/views/onboarding/createSampleEventButton';
+
+interface FirstEventFooterProps {
+  handleFirstIssueReceived: () => void;
+  hasFirstEvent: boolean;
+  isLast: boolean;
+  onClickSetupLater: () => void;
+  organization: Organization;
+  project: Project;
+}
+
+export default function FirstEventFooter({
+  organization,
+  project,
+  onClickSetupLater,
+  isLast,
+  hasFirstEvent,
+  handleFirstIssueReceived,
+}: FirstEventFooterProps) {
+  const source = 'targeted_onboarding_first_event_footer';
+
+  const getSecondaryCta = ({firstIssue}: {firstIssue: null | true | Group}) => {
+    // if hasn't sent first event, allow creation of sample error
+    if (!hasFirstEvent) {
+      return (
+        <CreateSampleEventButton
+          project={project}
+          source="targted-onboarding"
+          priority="default"
+        >
+          {t('View Sample Error')}
+        </CreateSampleEventButton>
+      );
+    }
+    // if last, no secondary cta
+    if (isLast) {
+      return null;
+    }
+    return (
+      <Button
+        to={`/organizations/${organization.slug}/issues/${
+          firstIssue !== true && firstIssue !== null ? `${firstIssue.id}/` : ''
+        }`}
+      >
+        {t('Take me to my error')}
+      </Button>
+    );
+  };
+
+  const getPrimaryCta = ({firstIssue}: {firstIssue: null | true | Group}) => {
+    // if hasn't sent first event, allow skiping
+    if (!hasFirstEvent) {
+      return (
+        <Button priority="primary" onClick={onClickSetupLater}>
+          {t('Setup Later')}
+        </Button>
+      );
+    }
+    if (isLast) {
+      return (
+        <Button
+          to={`/organizations/${organization.slug}/issues/${
+            firstIssue !== true && firstIssue !== null ? `${firstIssue.id}/` : ''
+          }`}
+          priority="primary"
+        >
+          {t('Take me to my error')}
+        </Button>
+      );
+    }
+    return (
+      <Button priority="primary" onClick={onClickSetupLater}>
+        {t('Next Platform')}
+      </Button>
+    );
+  };
+
+  return (
+    <Wrapper>
+      <SkipOnboardingLink
+        onClick={() =>
+          trackAdvancedAnalyticsEvent('growth.onboarding_clicked_skip', {
+            organization,
+            source,
+          })
+        }
+        to={`/organizations/${organization.slug}/issues/`}
+      >
+        {t('Skip Onboarding')}
+      </SkipOnboardingLink>
+      <EventWaiter
```

Comment:
@Neo-Zhixing The problem is that I would have to pass `firstIssue` from the parent component down to this one since we need to link to the issue. So no matter what I have to pass a prop down so I don't see moving this to the parent to be an improvment.